BossMod/Autorotation/UIRotationWindow.cs

```diff
@@ -90,13 +92,15 @@ public static bool DrawRotationSelector(RotationModuleManager mgr)
 
         using (ImRaii.PushColor(ImGuiCol.Button, 0xff000080, mgr.Preset == RotationModuleManager.ForceDisable))
         {
-            if (ImGui.Button("X"))
+            if (ImGui.Button("Disable"))
             {
                 mgr.Preset = mgr.Preset == RotationModuleManager.ForceDisable ? null : RotationModuleManager.ForceDisable;
                 modified |= true;
             }
         }
 
+        ImGui.TextUnformatted("Presets:");
```

Comment:
Can we make it on same line as disable / X ? I'm somewhat conscious of vertical space taken by this window.